### ISSUE TITLE
how do I use this to select responses?

### ISSUE THREAD

**Issue by cristianmtr** [taesunwhang/BERT-ResSel#2]
Hey

How can I use this model to select responses?

**Comment by taesunwhang** [comment]
Hi, if you want to use this model for selecting responses, you need to make a set of response candidates.

**Comment by cristianmtr** [comment]
Yes, I got that. I'm just confused about what the API would be. I basically have a history of a conversation, and want to select the most appropriate response from a list of possible responses. 

Any tips? :)
